js/electron.js

```diff
@@ -31,6 +31,8 @@ function createWindow() {
 		}
 	}
 
+	var electronOptions = Object.assign({}, electronOptionsDefaults, config.electronOptions);
```

Comment:
Maybe move this after line 41, so that user-defined options override `kiosk`, `fullscreen` and `autohidemenubar`? Otherwise it isn't transparent that those specific options will be overridden by code.